tool/planet/udev.go

```diff
@@ -32,7 +32,7 @@ import (
 
 // newUdevListener creates a new udev event listener listening
 // for events on block devices of type `disk`
-func newUdevListener() (*udevListener, error) {
+func newUdevListener(seLinux bool) (*udevListener, error) {
```

Comment:
Random thought: I wonder if you considered making this a global parameter somewhere that would be initialized e.g. in an init() method, rather than having to pass it everywhere.

Reply:
Not sure I'd fancy anything like that - I think it's better when it's explicit even if a bit repetitive.